examples/checkPars.py

```diff
@@ -0,0 +1,162 @@
+#!/usr/bin/env python
+
+#
+# LSST Data Management System
+# Copyright 2015 AURA/LSST
+#
+# This product includes software developed by the
+# LSST Project (http://www.lsst.org/).
+#
+# This program is free software: you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.    See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the LSST License Statement and
+# the GNU General Public License along with this program.  If not,
+# see <http://www.lsstcorp.org/LegalNotices/>.
+#
+import argparse
+import math
+import sys
+import os.path
+import numpy
+import pyfits
+
+import lsst.daf.persistence as dafPersist
+import lsst.afw.coord as afwCoord
+import lsst.afw.cameraGeom as cameraGeom
+import lsst.afw.detection as afwDetection
+import lsst.afw.geom as afwGeom
+import lsst.afw.image as afwImage
+import lsst.afw.math as afwMath
+import lsst.afw.table as afwTable
+import lsst.meas.algorithms as measAlg
+import lsst.meas.base as measBase
+
+"""
+checkPars.py is a python program which can be called as a command line script.
+It is used to print the distribution of a library over the entire
+LSST focal plane, divided into groups (currently by raft)
+"""
+
+#   This routine takes a list of images and a catalog name (presumably covering the
```

Comment:
Documentation for the methods should go inside the methods formatted so that each input and output variable is described.